client/components/domains/free-domain-explainer/index.jsx

```diff
@@ -45,19 +45,25 @@ class FreeDomainExplainer extends React.Component {
 	}
 
 	render() {
-		const { translate, isReskinned } = this.props;
+		const { translate, isReskinned, locale } = this.props;
 		const { TextWrapper } = this;
 
 		return (
 			<div className="free-domain-explainer card is-compact">
 				<header>
 					<h1 className="free-domain-explainer__title">
-						{ translate( 'Get a free one-year domain registration with any paid plan.' ) }
+						{ locale === 'en'
```

Comment:
Just a note that this will not apply the change for en_gb locale, but en_gb users in US will still get assigned to the test. I don't know how many in the US will use en_gb though.

Reply:
The test is currently US-only, and the string will be translated, so this if will be removed at some point. I think we'll be fine, as long as we don't expand to another English-speaking country before we remove the if.

Out of curiosity, what would be a correct implementation if we had to support en_GB? The value here would be `en-gb`.

Reply:
Sounds good 👍 .

> Out of curiosity, what would be a correct implementation if we had to support en_GB? 

Here's an [example](https://github.com/Automattic/wp-calypso/blob/master/client/me/concierge/book/info-step.js#L132).